README.md

```diff
@@ -18,23 +28,23 @@ This release version of the library has made a few changes to standardize its us
 ## Documentation
 
 * See the tango documentation [here](http://tango-controls.readthedocs.io/en/latest/administration/services/hdbpp/index.html#hdb-an-archiving-historian-service) for broader information about the HB++ archiving system and its integration into Tango Controls
-* Libhdbpp-Cassandra [CHANGELOG.md](https://github.com/tango-controls-hdbpp/libhdbpp/blob/master/CHANGELOG.md) contains the latest changes both released and in development.
+* Libhdbpp-Cassandra [CHANGELOG.md](CHANGELOG.md) contains the latest changes both released[and in development.]
 
-## Bugs Reports
+## Bugs R)eports
```

Comment:
Why this parenthesis?

Reply:
Some kind of alien artifact has invaded my README....